timeseries/src/main/scala/com/criteo/cuttle/timeseries/TimeSeriesScheduler.scala

```diff
@@ -730,32 +730,47 @@ private[timeseries] object TimeSeriesUtils {
     */
   def validate(workflow: Workflow[TimeSeries]): Either[List[String], Unit] = {
```

Comment:
Would it be possible to expose this on Workflow ? use case is to validate a workflow externally before sending it to cuttle.

Reply:
Yes sure will do